com.termius.termius.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2019 Termius Corporation.> -->
+<component>
+ <id>com.termius.termius</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>LicenseRef-proprietary</project_license>
+ <name>Termius</name>
+ <summary>Termius is the SSH client that works on Desktop and Mobile</summary>
+ <description>
+  <p>
+  Securely Sync Data Across All Your Devices
+  Termius lets you organize hosts into groups.
+  Groups allow you to share settings, though each host can have its own separate preferences.
+  This data, along with connection and command history, is securely synced across all your devices.
+  Termius uses end-to-end encryption to ensure your data remains safe and secure.
+  </p>
+ </description>
```

Comment:
Add `<releases>` tag with version info to the appdata.

Reply:
Ok many thank for your great help :D